src/nls/root/strings.js

```diff
@@ -418,6 +418,7 @@ define({
     "CMD_SHOW_IN_EXPLORER"                : "Show in Explorer",
     "CMD_SHOW_IN_FINDER"                  : "Show in Finder",
     "CMD_SHOW_IN_OS"                      : "Show in OS",
+    "CMD_SPLITVIEW_TOGGLE"                : "Toggle Panes",
```

Comment:
Can we write it like this?
`"CMD_SHUFFLE_FOCUS_ACROSS_PANES"           : "Shuffle Focus Across Panes",`